tests/cpp/utils.cpp

```diff
@@ -43,6 +47,9 @@ void NVFuserTest::SetUp() {
 }
 
 void NVFuserTest::TearDown() {
+  at::cuda::clearCublasWorkspaces();
```

Comment:
This is not strictly required -- I can also clearCublasWorkspaces only in RuntimeTest.ClearGmemBetweenSegments. But I think it's nice to verify no GPU memory leaks for all tests.